hymaintenance/high_ui/static/js/fast_click.js

```diff
@@ -0,0 +1,19 @@
+function ForbiddenFastClick (document) {
+    var buttons = document.querySelectorAll('form button[type="submit"]');
+    for (var button of buttons) {
+        button.addEventListener("click", function (e) {
+            if (!button.classList.contains('is-form-submitting')) {
+                button.classList.add('is-form-submitting');
+            } else {
+                e.preventDefault();
+            }
+        })
+    }
+
+    forms = document.querySelectorAll("form");
+    for (var form of forms) {
+        form.addEventListener("submit", function (e) {
+            form.querySelector('form button[type="submit"]').classList.add('is-form-submitting');
```

Comment:
humm.. pour ce cas je sais pas si mettre 'is-form-submitting' au niveau du <form> serait pas plus sûr.

Reply:
j'en parlais aussi

Reply:
Je suis désolée mais je ne comprends pas de quel cas d'utilisation du form vous parlez. On peut submit le form sans cliquer sur le bouton submit ?

Reply:
oui par les users selon les champs en tapant sur entrée, ou avec du code en appelant la fonction submit du form en js

Reply:
oki je vois